www/content/extensions/server-sent-events.md

```diff
@@ -66,58 +66,55 @@ data: <div>Content to swap into your HTML page.</div>
 <div hx-ext="sse" sse-connect="/server-url" sse-swap="event1,event2"></div>
 
 Multiple events in different elements (from the same source).
 <div hx-ext="sse" sse-connect="/server-url">
-    <div sse-swap="event1"></div>
-    <div sse-swap="event2"></div>
+  <div sse-swap="event1"></div>
+  <div sse-swap="event2"></div>
 </div>
 ```
 
 ### Trigger Server Callbacks
 
-When a connection for server sent events has been established, child elements can listen for these events by using the special [`hx-trigger`](@/attributes/hx-trigger.md) syntax `sse:<event_name>`.  This, when combined with an `hx-get` or similar will trigger the element to make a request.
+When a connection for server sent events has been established, child elements can listen for these events by using the special [`hx-trigger`](@/attributes/hx-trigger.md) syntax `sse:<event_name>`. This, when combined with an `hx-get` or similar will trigger the element to make a request.
 
 Here is an example:
 
 ```html
-  <div hx-ext="sse" sse-connect="/event_stream">
-    <div hx-get="/chatroom" hx-trigger="sse:chatter">
-      ...
-    </div>
-  </div>
+<div hx-ext="sse" sse-connect="/event_stream">
+  <div hx-get="/chatroom" hx-trigger="sse:chatter">...</div>
+</div>
 ```
 
 This example establishes an SSE connection to the `event_stream` end point which then triggers
 a `GET` to the `/chatroom` url whenever the `chatter` event is seen.
 
 ### Automatic Reconnection
 
-If the SSE Event Stream is closed unexpectedly, browsers are supposed to attempt to reconnect automatically.  However, in rare situations this does not work and your browser can be left hanging.  This extension adds its own reconnection logic (using an [exponential-backoff algorithm](https://en.wikipedia.org/wiki/Exponential_backoff)) on top of the browser's automatic reconnection, so that your SSE streams will always be as reliable as possible.
+If the SSE Event Stream is closed unexpectedly, browsers are supposed to attempt to reconnect automatically. However, in rare situations this does not work and your browser can be left hanging. This extension adds its own reconnection logic (using an [exponential-backoff algorithm](https://en.wikipedia.org/wiki/Exponential_backoff)) on top of the browser's automatic reconnection, so that your SSE streams will always be as reliable as possible.
 
 ### Testing SSE Connections with the Demo Server
 
-Htmx includes a demo SSE server written in Go that will help you to see SSE in action, and begin bootstrapping your own SSE code.  It is located in the /test/servers/sse folder of the htmx distribution.  Look at /test/servers/ws/README.md for instructions on running and using the test server.
+Htmx includes a demo SSE server written in Node.js that will help you to see SSE in action, and begin bootstrapping your own SSE code. It is located in the /test/ws-sse folder of the htmx distribution. Look at /test/ws-sse/README.md for instructions on running and using the test server.
```

Comment:
I think it's just these two lines that we need.

Reply:
Fixed.